package.json

```diff
@@ -150,10 +150,8 @@
     "test:e2e-up": "./tests/e2e/bin/up.sh",
     "test:e2e-down": "./tests/e2e/bin/down.sh",
     "test:e2e-cleanup": "npm run test:e2e-down && ./tests/e2e/bin/cleanup.sh",
-    "test:e2e": "./tests/e2e/bin/run-tests.sh --project=default",
-    "test:e2e-debug": "npm run test:e2e -- --debug",
-    "test:e2e-legacy": "./tests/e2e/bin/run-tests.sh --project=legacy",
-    "test:e2e-legacy-debug": "npm run test:e2e-legacy -- --debug"
+    "test:e2e": "./tests/e2e/bin/run-tests.sh",
+    "test:e2e-debug": "npm run test:e2e -- --debug"
```

Comment:
Why are we reverting to using one E2E test suite?

Reply:
It's not part of this PR; reverted it.